beautiful-and-accessible-drag-and-drop-with-react-beautiful-dnd-notes/10-react-conditionally-allow-movement-using-react-beautiful-dnd-draggable-and-droppable-props.md

```diff
@@ -0,0 +1,176 @@
+## :movie_camera: [Lesson 10](https://egghead.io/lessons/react-conditionally-allow-movement-using-react-beautiful-dnd-draggable-and-droppable-props)
+
+
+<TimeStamp start="0:01" end="0:22">
+
+There is basically an on/off switch to components to  to configure what can be dragged or dropped where. To do this, we use `isDragDisabled`
+
+```js
+<Draggable 
+  draggableId = {this.props.task.id} 
+  index={this.props.index}
+  isDragDisabled={true}
+>
+```
+
+</TimeStamp>
+
+<TimeStamp start="0:47" end="0:50">
+
+We can make it so that there are conditions to wether or not a task can be dragged. We will make a new `const` and assign our conditions to that.
```

Comment:
Great note. Adds context the instructor didn't give

Reply:
```suggestion
We can make it so that there are conditions to whether or not a task can be dragged. We will make a new `const` and assign our conditions to that.
```